client/src/Components/Flows/AuthenticationFlow/Elements/RegistrationForm.js

```diff
@@ -15,13 +17,17 @@ export const RegistrationForm = (props) => {
       email,
       password,
     })
-    if (props.user.name) {
-      props.user.accountType === 'sender' ? 
+  }
+
+  useEffect(() => {
+    // Used to evaluate if Registration and the nested Login and GetUser have been successful and a user object has been returned from the back-end
+    if (user.email) {
+      user.accountType === 'sender' ? 
         props.goToSelectAmount()
-          : 
-        props.pushToUserProfile()
+      : 
+        props.goToWallet()
     }
-  }
+  });
```

Comment:
With your comment below "Thinking of using react-dropdown..." I say just use native `<select>` elements first and then we can refactor later.

Reply:
Good idea! Will add to the menu card on Trello!